### PR TITLE
implemented hamburger component in dashboard for desktop view

### PR DESCRIPTION
**Pull request by capsule11**
UI Changes in home page
Added Hamburger component for admin dashboard in desktop view

### CODE REVIEW

**Comment by Sreejit-Sengupto** [comment]
⚠️ MERGE NAHI KARNA HAI ISKO ABHI!!

**Comment by capsule11** [comment]
Hn abhi iss code me bahut ulta seedha pata nhi kya kya hua hai check karke merge karna

File: src/pages/admin/dashboard/dashboard.jsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const DashboardPage = () => {
   return (
     <div className="w-full h-[100dvh] flex flex-col">
       <TopBar category={"ORG"} />
-      <div className="grid grid-cols-6 h-full">
```
Comment (Sreejit-Sengupto) [inline src/pages/admin/dashboard/dashboard.jsx:24]: 🦥

**Review by Sreejit-Sengupto — CHANGES_REQUESTED** [review]

File: src/pages/admin/dashboard/dashboard.jsx (hunk 1 above)
Comment (capsule11) [reply src/pages/admin/dashboard/dashboard.jsx:24]: ?

Comment (capsule11) [reply src/pages/admin/dashboard/dashboard.jsx:24]: image
see

**Pull request closed without merging**